sdk/core/System.ClientModel/tests/client/Internal/ModelReaderWriterExtensions.cs

```diff
@@ -0,0 +1,264 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please move all of these into a subfolder of ModelReaderWriter.  You can follow the example in the feature branch https://github.com/Azure/azure-sdk-for-net/pull/39135/files#diff-222df6462017558f59f72215ae9c867458a10f8ab4724984ef8b02e3b46626b6R1